storm-netty/src/jvm/backtype/storm/messaging/netty/Context.java

```diff
@@ -59,9 +59,9 @@ public IConnection connect(String storm_id, String host, int port) {
      * terminate this context
      */
     public void term() {
-        for (IConnection conn : connections) {
+        connections.forEach(conn -> {
```

Comment:
@desmorto, You can remove unnecessary brackets or use a method reference.
<pre>
connections.forEach(conn -> conn.close());
</pre>